src/victoria_metrics.c

```diff
@@ -127,12 +150,12 @@ void * vm_push(void * arg) {
 
         curl_easy_setopt(curl, CURLOPT_POSTFIELDSIZE, buffer_size);
         curl_easy_setopt(curl, CURLOPT_POSTFIELDS, buffer);
+        curl_easy_setopt(curl, CURLOPT_FAILONERROR, 1);
         res = curl_easy_perform(curl);
         if (res != CURLE_OK) {
             printf("Failed push: %s\n", curl_easy_strerror(res));
-        } else {
-            buffer_size = 0;
```

Comment:
Why this change?
Before I had this so that a pc with connection drop could use up the buffer until connection is back

Reply:
I think I did that so that debug prints wouldn't flood the output while I was testing the Docker version of Victoria Metrics. But now that this is in place, there's no need to actually change the current behaviour, I'll put it back in as it was.